metabase/metabase
# Distinguish multiple "Summaries" column groups in dashboard filter mapping

Closes #48260

Depends on #48254

### How to verify

1. Create a query with aggregations in at least 2 stages ([example](https://github.com/user-attachments/assets/d9d1be0a-7a60-40e0-b065-8148bfc3b180)) and add it to a dashboard
2. Columns from different summary stages should be distinguishable

CI won't be green - 1 test is failing because of #48258

### Demo

[before](https://github.com/user-attachments/assets/df163164-2768-4a53-8a42-03f1b917aa11) vs [after](https://github.com/user-attachments/assets/a51203e3-310f-460c-80a5-d99963189c51)
